### ISSUE TITLE
Dynamic economic model parameters

### ISSUE THREAD

**Issue by jeandemeusy** [hoprnet/ct-research#476]
TLDR
Someone who manually staked 75k would soon be above the economic model linear threshold because of the rewards he received. In following distributions, this node runner would not see his rewards increase as much as his stake increased.

Then he might think about running an additional node to split his stake by two and have every single wxHOPR taken into account by CT. However, this has two drawbacks:
- Spinning an additional node would increase stress on the network
- The cost of running an additional node would certainly not be covered by the small additional rewards received. 

If the cost of running a node is not covered by APR, then the node runner might think of withdrawing the extra wxHOPR from their safe, which HOPR wants to avoid.

See proposal here: https://docs.google.com/document/d/10apy630yE61b2tKBHjqZPYXpE5NXPlH67-eUlQCMajU/edit#heading=h.lpkq3992g6s

**Comment by jeandemeusy** [comment]
Some work needs to be done for the last point of the proposal: do some research on which subgraph to use to get the RedeemedEvent from on-chain, otherwise proposal can be already discussed.

**Comment by mjadach-iv** [comment]
I would keep the 75k myself, and let people add more nodes as it will not bringt that many new nodes to the network and every node counts.
You can also try to make it upgradable to 80k after 1 month of successful distrubutions to a safe. We could then show status the 75k/80k uprade in the safe ui.

**Comment by jeandemeusy** [comment]
Done in https://github.com/hoprnet/ct-research/pull/491
